tests/puppeteer/pages/BO/design/pages/pages.js

```diff
@@ -0,0 +1,363 @@
+    this.pageGridTitle = `${this.pageGridPanel} h3.card-header-title`;
+    this.pagesListForm = '#cms_page_grid';
+    this.pagesListTableRow = `${this.pagesListForm} tbody tr:nth-child(%ROW)`;
+    this.pagesListTableColumn = `${this.pagesListTableRow} td.column-%COLUMN`;
+    this.pageListTableToggleDropDown = `${this.pagesListTableColumn} a[data-toggle='dropdown']`;
+    this.pagesListTableEditLink = `${this.pagesListTableColumn} a[href*='edit']`;
+    this.pagesListTableDeleteLink = `${this.pagesListTableColumn} a[data-method="DELETE"]`;
+    this.pagesListColumnValidIcon = `${this.pagesListTableColumn} i.grid-toggler-icon-valid`;
+    this.pagesListColumnNotValidIcon = `${this.pagesListTableColumn} i.grid-toggler-icon-not-valid`;
+
+    // Filters in pages table
+    this.pageFilterInput = `${this.pagesListForm} #cms_page_%FILTERBY`;
+    this.pageFilterSearchButton = `${this.pagesListForm} button[name='cms_page[actions][search]']`;
+    this.pagefilterResetButton = `${this.pagesListForm} button[name='cms_page[actions][reset]']`;
+    // Bulk Actions
+    this.pagesSelectAllRowsLabel = `${this.pagesListForm} .md-checkbox label`;
+    this.pagesBulkActionsToggleButton = `${this.pagesListForm} button.dropdown-toggle`;
+    this.pagesBulkActionsEnableButton = `${this.pagesListForm} #cms_page_grid_bulk_action_enable_selection`;
+    this.pagesBulkActionsDisableButton = `${this.pagesListForm} #cms_page_grid_bulk_action_disable_selection`;
+    this.pagesBulkActionsDeleteButton = `${this.pagesListForm} #cms_page_grid_bulk_action_delete_bulk`;
+  }
+
+  /*
+ Methods
+  */
+
+  /**
+   * Reset input filters
+   * @return {Promise<void>}
+   */
+  async resetFilter(selector) {
+    await Promise.all([
+      this.page.waitForNavigation({waitUntil: 'networkidle0'}),
+      this.page.click(selector),
+    ]);
+  }
+
+  // Methods for categories
+  /**
+   * Filter list of categories
+   * @param filterType, input or select to choose method of filter
+   * @param filterBy, column to filter
+   * @param value, value to filter with
+   * @return {Promise<void>}
+   */
+  async filterCategories(filterType, filterBy, value = '') {
+    switch (filterType) {
+      case 'input':
+        await this.setValue(this.categoryFilterInput.replace('%FILTERBY', filterBy), value);
+        break;
+      case 'select':
+        await this.selectByVisibleText(this.categoryFilterInput.replace('%FILTERBY', filterBy), value ? 'Yes' : 'No');
+        break;
+      default:
+      // Do nothing
+    }
+    // click on search
+    await Promise.all([
+      this.page.waitForNavigation({waitUntil: 'networkidle0'}),
+      this.page.click(this.categoryFilterSearchButton),
+    ]);
+  }
+
+  /**
+   * Get Value of column Displayed in Categories table
+   * @param row, row in table
+   * @param column, column to check
+   * @return {Promise<boolean|true>}
+   */
+  async getToggleColumnValueCategory(row, column) {
+    if (await this.elementVisible(
```

Comment:
replace whole function content by
```suggestion
  return await this.elementVisible(
    this.categoriesListColumnValidIcon
        .replace('%ROW', row)
        .replace('%COLUMN', column),
    100
  );
```